app/controllers/openingtimes_controller.rb

```diff
@@ -48,10 +51,10 @@ def update
   # DELETE /openingtimes/1 or /openingtimes/1.json
   def destroy
     @openingtime.destroy
-    respond_to do |format|
-      format.html { redirect_to openingtimes_url, notice: "Openingtime was successfully destroyed." }
-      format.json { head :no_content }
-    end
+    #     respond_to do |format|
+    #       format.html { redirect_to openingtimes_url, notice: "Openingtime was successfully destroyed." }
+    #       format.json { head :no_content }
```

Comment:
Why is this commented out? Do we still need it somewhere down the line or can this be removed for good?